src/main/java/org/jabref/gui/preferences/AppearanceTabViewModel.java

```diff
@@ -81,10 +81,10 @@ public void storeSettings() {
         }
 
         if (themeLightProperty.getValue() && !preferences.get(JabRefPreferences.FX_THEME).equals(ThemeLoader.MAIN_CSS)) {
-            restartWarnings.add(Localization.lang("Theme changed:") + " " + ThemeLoader.MAIN_CSS);
```

Comment:
I would include the theme name directly in the localization: `Localization.lang("Theme changed to light theme");`. This way, the theme name is also translated. I would accept the two new localization strings.